src/main/java/calculator/model/Calculator.java

```diff
@@ -0,0 +1,19 @@
+package calculator.model;
+
+public class Calculator {
+    private final Numbers numbers;
+    private final Operators operators;
+
```

Comment:
```suggestion
public Calculator(List<Number> numbers, List<Operator> operators) {
    this(new Numbers(numbers), new Operators(operators));
}
```
이런 부 생성자도 추가한다면 Calculator을 생성하는 곳에서 타입 변환을 하지 않아도 되니 편리하지 않을까?
여러 생성자를 추가해 이 객체를 사용하는 개발자의 편의성을 도모해 보면 어떨까?